app/components/Layout.js

```diff
@@ -97,7 +97,7 @@ class Layout extends React.Component<Props> {
     return (
       <Container column auto>
         <Helmet>
-          <title>Outline</title>
+          <title>{team.name}</title>
```

Comment:
`team` might be undefined in signed out situations – this will cause an error and must be guarded against. 

I'd also like to keep the product name, "Outline" at the end of the title tag.